src/__tests__/custom-rules/custom-rules.spec.ts

```diff
@@ -119,6 +120,72 @@ describe("custom-rules", () => {
 		);
 	});
 
+	it.each`
```

Comment:
why not add these tests to the existing it.each test block in line 15?

Reply:
the data is set to `{ field: valid, field1: valid }`, the equivalent for `notEqualsField` would be something like `{ field: valid, field1: invalid }` which I felt would be confusing to keep int he same test